flink-runtime/src/main/java/org/apache/flink/runtime/util/bash/BashJavaUtils.java

```diff
@@ -39,6 +45,7 @@
  * Utility class for using java utilities in bash scripts.
  */
 public class BashJavaUtils {
+	private static final Logger LOG = LoggerFactory.getLogger(BashJavaUtils.class);
```

Comment:
We cannot rely on external logging in `BashJavaUtils`.
The logs have to be forced into the stdout independently from logging config to pass them to `FLINK_INHERITED_LOGS` env var which is used by the main Flink process to log on its startup.

Reply:
It works though? All logging output goes to stdout, and the configuration things are split out based on a prefix. Everything else is treated as logging output; see `flink-dist/src/main/flink-bin/bin/config.sh:537`

Reply:
oh, we actually rely on default log config to output to stdout, sorry for confusion